desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py

```diff
@@ -344,22 +344,13 @@ def test_dialect_trim_statement_semicolon(self):
             execute=execute
           )
           notebook = {}
-          snippet = {'statement': 'SELECT 1;'}
+          snippet = {'statement': 'SELECT 1;', 'result': {}}
 
           # Trim
           engine = SqlAlchemyApi(self.user, interpreter).execute(notebook, snippet)
 
           execute.assert_called_with('SELECT 1')
 
-          # No Trim
-          interpreter['options']['url'] = 'mysql://hue:3306/hue'
-          interpreter['dialect_properties']['trim_statement_semicolon'] = False
-          interpreter['dialect_properties']['sql_identifier_quote'] = '`'
-
-          engine = SqlAlchemyApi(self.user, interpreter).execute(notebook, snippet)
-
-          execute.assert_called_with('SELECT 1;')
-
```

Comment:
Why remove the full test?

Reply:
Please keep the test.., just update it

Reply:
done, just removing the no trim part because now we don't have a property like 'trim_statement_semicolon'

Reply:
Should just remove the `          interpreter['dialect_properties']['trim_statement_semicolon'] = False` line?

Reply:
done, but also as we don't have the option to choose to no trim so need to update the last assert statement without semicolon.